Documentation/git-bash.txt

```diff
@@ -0,0 +1,51 @@
+git-bash(0)
+==========
+
+NAME
+----
+git-bash - A Windows initialiser for the bash window on that OS
+
+SYNOPSIS
+--------
+
+git-bash [--cd-to-home] [--xx] []
```

Comment:
I am fairly sure that `--xx` is not a valid option.

Reply:
I wasn't sure what the valid options are...